src/harness/reference_models/iap/test_data/cbsd_0.json

```diff
@@ -0,0 +1,44 @@
+{
+        "_comment":"FSS - Inband Blocking CBSD partial overlap",
+	"id": "device_0_id",
+	"registrationRequest": {
+					"fccId": "test_fcc_id_a",
+		            "cbsdCategory": "B",
```

Comment:
Change the CBSD category to A or change the indoorDeployment value to false

Reply:
Updated